nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -100,24 +100,24 @@ ch_output_docs_images = file("$baseDir/docs/images/", checkIfExists: true)
 /*
  * Create a channel for input read files
  */
-if (params.readPaths) {
+if (params.input_paths) {
```

Comment:
Ugh, all this code is nasty 😞 

One day... https://github.com/nf-core/tools/issues/211

Reply:
Yeah, I know... I bypass it all using design files as `--input` 😆 Not nice though, agreed.